src/main/sensors/rpm_filter.c

```diff
@@ -28,11 +28,12 @@
 #include "flight/mixer.h"
 #include "flight/pid.h"
 #include "pg/pg_ids.h"
+#include "scheduler/scheduler.h"
 #include "sensors/rpm_filter.h"
 #include "sensors/gyro.h"
 
 #define RPM_FILTER_MAXHARMONICS 3
-#define RPM_MOTOR_FILTER_CUTOFF 150
+#define RPM_MOTOR_FILTER_CUTOFF 250
```

Comment:
can we expose this as a parameter, or take it out of this change?

Reply:
I turned it into a parameter, @fujin